### PR TITLE
Updated pagination + social share

### PR DESCRIPTION
**Pull request by dellsystem**
Following discussion in PR #53, the social share & pagination buttons have been split up into separate lines.

Here's how it looks at the bottom of the page:
image

I kept the pagination at the top of the page, as I personally find it very useful when browsing, and I think it could be helpful to anyone trying to get a sense of what topics have been covered in the past without wanting to read the whole thing (eg, contributors). The social share buttons are only at the bottom.

I didn't have the energy to get the copy link button working - it's unfortunately not that trivial because different browsers implement it differently - but if anyone wants to add that in real quick, go ahead. Otherwise, I think we're fine without it, as people can just use their browser's tool for that. I also omitted the email button, as anyone reading in an email client will be able to use their client's native forwarding functionality.

### CODE REVIEW

**Comment by stationaery** [comment]
Having pagination at the top seems OK to me. It might even help give more
attention to the share links at the bottom.

For a 'copy link' button, is there an easy, straightforward solution using
Web Share https://w3c.github.io/web-share/? I browsed
https://dvcs.w3.org/hg/web-intents/raw-file/tip/spec/Overview.html and it
looks like the ability to trigger the browser share functionality might be
as simple as a single URL. (Personally, I almost alway grab the link and
then tweet/post/email in order to avoid the awkward portal in which I have
to compose the rest of the text, and also to avoid any UTM code).

File: _includes/social_share.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<nav aria-label="Share on social media">
+    <ul class="pagination justify-content-center">
+        <li class="page-item">
+            <a class="page-link" href="https://twitter.com/intent/tweet?text={{ page.title|url_encode }}%20via%20@twcnewsletter%20{{ site.url }}{{ page.url }}" title="Share on Twitter" target="_blank"><i class="fa fa-twitter"></i> Tweet</a>
+        </li>
+        <li class="page-item">
+            <a class="page-link" href="https://www.facebook.com/sharer.php?u={{ site.url }}{{ page.url }}" title="Share on Facebook" target="_blank"><i class="fa fa-facebook"></i> Post</a>
```
Comment (jessesquires) [inline _includes/social_share.html:7]: I *think* we can use `{{ page.url | absolute_url }}` instead here

**Comment by jessesquires** [comment]
If we want 3 options, we could add "Subscribe" to give us `Tweet | Post | Subscribe`

**Comment by stationaery** [comment]
Pagination, sharing, and subscribing are fairly different functions that
sit awkwardly for me side-by-side 😬

File: _includes/social_share.html (hunk 1 above)
Comment (dellsystem) [reply _includes/social_share.html:7]: good point - i'll make that change

**Comment by dellsystem** [comment]
I'm going to merge this in for now. If someone wants to figure out the "Copy link" button, that could go in a new PR.

Closes #51!

**Comment by jessesquires** [comment]
🎉

**Pull request merged**